components/python/pycurl/history

```diff
@@ -1,4 +1,6 @@
-library/python/pycurl-34@7.43.0.2-2020.0.1.1
+library/python/pycurl-34@7.43.0.2,5.11-2020.0.1.1
 library/python-2/pycurl-26@7.19.0.1,5.11-2016.0.1.1
 library/python-2/pycurl-27@7.19.0.1,5.11-2016.0.1.1 library/python/pycurl-27
```

Comment:
You should obsolete this package too because you should never have obsoleted rename target.